### PR TITLE
feat: Minimizable Dialog updates

### PR DESCRIPTION
**Pull request by shleewhite**
- [x] update styles to one twilio
- [x] fix border radius so background color doesn't show up in corners of the dialog
- [x] add toggleMinimized function to the useMinimizableDialogState return to match the toggle function that comes from the useNonModalDialogPrimitiveState hook
- [x] break out the styledMinimizableDialog component out so that it can be used in VRT (similar to how the modal VRT stories work)
- [x] update the default and default minimized stories so theyre not all stacked on top of each other

**Contributing to Twilio**

> All third-party contributors acknowledge that any contributions they provide will be made under the same open-source license that the open-source project is provided under.

- [ ] I acknowledge that all my contributions will be made under the project's license.

### CODE REVIEW

**Review by TheSisb — DISMISSED** [review]

**Review by TheSisb — DISMISSED** [review]

**Review by nkrantz — APPROVED** [review]

**Review by loreina — DISMISSED** [review]

File: packages/paste-core/components/minimizable-dialog/src/StyledMinimizableDialog.tsx [hunk 1]
```diff
@@ -0,0 +1,21 @@
+import * as React from 'react';
+import {Box} from '@twilio-paste/box';
+
+export const StyledMinimizableDialog: React.FC<{element?: string; children: React.ReactNode}> = ({
```
Comment (SiTaggart) [inline packages/paste-core/components/minimizable-dialog/src/StyledMinimizableDialog.tsx:4]: issue(blocker): This needs to be changed for React18. We can no longer do `React.FC` typings without `React.PropsWithChildren`

```
React.FC<React.PropsWithChildren<AvatarContentProps>>
```

Comment (SiTaggart) [reply packages/paste-core/components/minimizable-dialog/src/StyledMinimizableDialog.tsx:4]: Side note, worries me it doesn't fail type checking

Comment (shleewhite) [reply packages/paste-core/components/minimizable-dialog/src/StyledMinimizableDialog.tsx]: ooohh is this a react 18 thing?

**Review by loreina — APPROVED** [review]

File: packages/paste-core/components/minimizable-dialog/src/StyledMinimizableDialog.tsx (hunk 1 above)
Comment (SiTaggart) [reply packages/paste-core/components/minimizable-dialog/src/StyledMinimizableDialog.tsx]: Yup Yup. I have updated the VS code snippet too, but I don't know if anyone uses them

**Pull request merged**